CHANGELOG.md

```diff
@@ -6,6 +6,15 @@ _The format is based on [Keep a Changelog](http://keepachangelog.com/) and this
 
 ## [unreleased]
 
+## [v0.6.0] - 2024-09-18
```

Comment:
This version change entry should be postponed to when we create the release - just leave it under `unreleased` for now